apps/backend/src/deposits/cash-deposit.service.ts

```diff
@@ -93,11 +93,8 @@ export class CashDepositService {
     const distinctDepositDates = Array.from(
       new Set(dates.map((item) => item.deposit_date))
     );
-    if (distinctDepositDates.length < 4)
-      return {
-        currentDate: date,
-        pastDueDate: null
-      };
+    if (distinctDepositDates.length < 4) return;
```

Comment:
Don't throw exceptions if we have less than 4 deposit dates of data to look back to

Reply:
(should we make this 3?)